test/test_rules.jl

```diff
@@ -247,6 +374,28 @@ layers = Dict(
     end
 end
 
+## Test Scale layer
+# Define Scale test input
+d = 4 # input + output dimension
+batchsize = 2
+aᵏ_dense = pseudorandn(d, batchsize)
+
+layers = Dict(
+    "Scale_relu"     => Scale(pseudorandn(d), pseudorandn(d), relu),
+    "Scale_identity" => Scale(ones(Float32, d), false, identity),
+)
+@testset "Scale" begin
+    for (rulename, rule) in RULES
+        @testset "$rulename" begin
+            for (layername, layer) in layers
+                @testset "$layername" begin
+                    run_rule_tests(rule, layer, rulename, layername, aᵏ_dense)
+                end
+            end
+        end
+    end
+end
+
```

Comment:
I tried to do these tests analogous to the `Dense` tests but I did not really know what I was doing - maybe you can check if this is actually usefull :D